models/heirbrant2023boekbazaar/metadata.yaml

```diff
@@ -0,0 +1,34 @@
+title: Boekbazaar Ontology
+acronym:
+issued: 2023
+modified:
+contributor:
+ - https://www.linkedin.com/in/joachim-heirbrant-532aa1233/
+ - https://dblp.org/pid/251/5991
+ - https://dblp.org/pid/95/6356
+keyword:
+ - digital platform ontology
```

Comment:
We use this field to identify domains, not for any keywords that may apply to the model.

Reply:
Done.